help/destinations/catalog/crm/salesforce.md

```diff
@@ -0,0 +1,202 @@
+
+### Schedule segment export and example {#schedule-segment-export-example}
+
+* When performing the [Schedule segment export](../../ui/activate/activate-segment-streaming-destinations.html?lang=en#scheduling) step you will need to manually map Platform segments to the custom field attribute in Salesforce.
+* To do this, select each segment, then enter the corresponding custom field attribute from Salesforce in the **[!UICONTROL Mapping ID]** field.
+
+    > [!IMPORTANT]
+    >
+    > * The value used for the Mapping ID should exactly match the name of the custom field attribute created within Salesforce.
+    > * Ensure that the name of the custom field attribute you have created in Salesforce doesn't use the whitespace character.
+
+* An example is shown below:
+![Schedule segment export](../../assets/catalog/crm/salesforce/schedule-segment-export.png)
+
+## Validate data export {#exported-data}
+
+To validate that you have correctly set up the destination, follow the steps below:
+
+1. Select **[!UICONTROL Destinations]** > **[!UICONTROL Browse]** to navigate to the list of destinations.
+![Browse Destinations](../../assets/catalog/crm/salesforce/browse-destinations.png)
+
+1. Select the destination and validate that the status is **[!UICONTROL enabled]**.
+![Destinations Dataflow Run](../../assets/catalog/crm/salesforce/destination-dataflow-run.png)
+
+1. Switch to the **[!DNL Activation data]** tab, then select a segment name.
+![Destinations Activation Data](../../assets/catalog/crm/salesforce/destinations-activation-data.png)
+
+1. Monitor the segment summary and ensure that the count of profiles corresponds to the count created within the segment.
+![Segment](../../assets/catalog/crm/salesforce/segment.png)
+
+1. Login to the Salesforce website, then navigate to the **[!DNL Apps]** > **[!DNL Contacts]** page and check if the profiles from the segment have been added.
+![Salesforce Contacts](../../assets/catalog/crm/salesforce/contacts.png)
+
+1. Click a contact and check if the fields are updated. You will notice the segment status from Experience Platform has been updated against the corresponding custom field attribute that was provided in the **Mapping ID** field during the **[!UICONTROL Activate destination]** > **[!UICONTROL Schedule segment export]** step.
+![Salesforce Contacts](../../assets/catalog/crm/salesforce/contact-info.png)
+
+## Data usage and governance {#data-usage-governance}
+
+All [!DNL Adobe Experience Platform] destinations are compliant with data usage policies when handling your data. For detailed information on how [!DNL Adobe Experience Platform] enforces data governance, see the [Data Governance overview](/help/data-governance/home.md).
+
+## Errors and troubleshooting {#errors-and-troubleshooting}
+
+When checking a dataflow run, if you obtain the below error message; Check that the Mapping ID you provided against your Platform segment is valid and exists within [!DNL Salesforce CRM].
```

Comment:
Minor comment here. Considering that maybe we'll want to add some more errors in the future, we could have a subheading for each error. So for the existing one, we would have: 

```
### Unknown errors encountered while pushing events to destination

When checking a dataflow run, if you see the error message below, verify that the **Mapping ID** that you provided in [!DNL Salesforce CRM] for your Platform segment is valid and exists within [!DNL Salesforce CRM].
```